src/Attribute/ReqAttribute.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\RequestModel\Attribute;
+
+use Attribute;
+
+#[Attribute(flags: Attribute::TARGET_PARAMETER | Attribute::IS_REPEATABLE)]
+final class ReqAttribute implements HandlerParameterAttributeInterface
```

Comment:
RequestAttribute?

Reply:
I'm OK with it. Just wanted to use the short names.

Reply:
Could be just Request.